kotlin-asyncapi-spring-web/src/test/kotlin/org/openfolder/kotlinasyncapi/springweb/context/external/DefaultAnnotationProviderTest.kt

```diff
@@ -35,13 +42,29 @@ internal class DefaultAnnotationProviderTest {
     @EnableAsyncApi
     open class TestConfig
 
+    @Channel(
+        value = "test/{parameter}/sqs",
```

Comment:
A bit confusing in the example value sqs, channel binding AMQP

Reply:
You are right. I changed it 👍